app/grandchallenge/reader_studies/views.py

```diff
@@ -335,6 +337,14 @@ def export_answers(self, request, pk=None):
             filename=f"{reader_study.slug}-answers.csv",
         )
 
+    @action(detail=True)
```

Comment:
Should this be a POST request as the reader_study is mutated?

Reply:
Yes, definitely.